app/controllers/observations_controller.rb

```diff
@@ -1,7 +1,8 @@
 class ObservationsController < ApplicationController
   before_action :authenticate_user!
   before_action :find_proposal
-  before_action ->{authorize @proposal, :can_show!}
+  before_action ->{authorize Observation.new(proposal: @proposal)}, only: :create
```

Comment:
Kinda wish there was a cleaner way to do this...

https://github.com/elabs/pundit#additional-context